modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/provider/LayoutInfoItemLanguagesProvider.java

```diff
@@ -52,6 +52,11 @@ public String getDefaultLanguageId(Layout layout) {
 			layout);
 	}
 
+	@Override
+	public String[] getInfoItemAvailableLanguageIds(Layout layout) {
+		return layout.getAvailableLanguageIds();
+	}
+
```

Comment:
Cannot we get rid of the other logic and simply return all layout languages? Shouldn't the layout include all the languages for all its fragments?